Modules/IO/ImageBase/test/itkLargeImageWriteReadTest.cxx

```diff
@@ -125,7 +125,7 @@ ActualTest(std::string filename, typename TImageType::SizeType size)
   ritr.GoToBegin();
   // oitr.GoToBegin();
 
-  std::cout << "Comparing the pixel values.. :" << std::endl;
+  std::cout << "Comparing the pixel values... :" << std::endl;
```

Comment:
Thanks Jon, but `... :` still looks a but unusual to me. Maybe just remove the ellipsis? LargeTIFFImageWriteReadTest does not have them either:

https://github.com/InsightSoftwareConsortium/ITK/blob/1971476979daa3fc6f633bd69fc9d197a6760771/Modules/IO/TIFF/test/itkLargeTIFFImageWriteReadTest.cxx#L130

Reply:
I agree. Having both ellipsis and colon is strange.

Reply:
I did have the same feeling when proposing the PR, but did not look further. Removed the colon.